CHANGELOG.rst

```diff
@@ -9,6 +9,7 @@ CHANGELOG
 * bug-fix: Local Mode: correctly handle the case where the model output folder doesn't exist yet
 * doc-fix: Fix typos in tensorflow serving documentation
 * doc-fix: Add estimator base classes to API docs
+* feature: SageMaker Automatic Model Tuning's Warm Start jobs integration
```

Comment:
imperative, commit message style:

Add support for Automatic Model Tuning Warm Start jobs or
Support Automatic Model Tuning Warm Start jobs or

Reply:
done